src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -4497,6 +4505,7 @@ INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','ActiveEffluxSpeci
 INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','ActiveEffluxSpecificWithTransporterInSource_MM','TM * kcat * K_water_src * C_src / (Km + K_water_src * C_src)','Amount per time');
 INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','ActiveInfluxSpecificWithTransporterInTarget_Hill','TM * kcat * (K_water_src * max(C_src;0))^alpha / (Km^alpha + (K_water_src * max(C_src;0))^alpha)','Amount per time');
 INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','ActiveInfluxSpecificWithTransporterInTarget_MM','TM * kcat * K_water_src * C_src / (Km + K_water_src * C_src)','Amount per time');
+INSERT INTO tab_calculation_method_rates VALUES('LinksCommon','ActiveBiDirectionalSpecific_MM','TM * kcat * (C_src * K_water_src - C_target * K_water_target) / (Km + C_src * K_water_src + C_target * K_water_target)','Amount per time');

```

Comment:
@PavelBal I don't know if the formula is correct. If you say "yes" - I'll just trust you :)